Dockerfile

```diff
@@ -4,9 +4,10 @@ RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 
 COPY . /go/src/metacontroller.app/
 WORKDIR /go/src/metacontroller.app/
+ENV CGO_ENABLED=0
 RUN dep ensure && go install
 
-FROM debian:stretch-slim
-RUN apt-get update && apt-get install --no-install-recommends -y ca-certificates && rm -rf /var/lib/apt/lists/*
+FROM alpine:3.12.0
```

Comment:
@grzesuav have you checked this file https://github.com/AmitKumarDas/metac/blob/master/Dockerfile#L66
We want to move towards gcr.io/distroless images. Most of the Kubernetes community makes use of this due to security reasons.

Reply:
@AmitKumarDas yes, I am aware, but it is a big change for user -  with small update you get image which you cannot exec into. That is why I wanted to introduce first alpine, and then distroless, but as a build variant, so that users can choose .

I imagine naming like :
* metacontrollerio/metacontroller:v4.0.1 == metacontrollerio/metacontroller:v4.0.1-alpine
* metacontrollerio/metacontroller:v4.0.1-distroless
so by default point to alpine, but if user want - it can use distroless one. 

I would be careful with making distroless default at it changes user experience a lot

Reply:
created issue for that https://github.com/metacontroller/metacontroller/issues/31